src/App.module.css

```diff
@@ -0,0 +1,16 @@
+.btnConfirm {
```

Comment:
Pleas ehave a look on this: https://getbem.com/
It will help you with the class naming approach